### DISCUSSION TITLE
Is the "Setting up the Environment" section in the CONTRIBUTING.md missing a step?

### DISCUSSION THREAD

**Discussion by gwhitney** [pnpm/pnpm#5288] [Q&A] [answered]
I forked and cloned the pnpm repository for the first time, and followed the instructions in "Setting up the Environment" in CONTRIBUTING.md exactly. However, in step 3, after changing a source file, when I executed `pd install` (or `pd --version` or any other pd command), I would always get the error `Could not resolve "/home/glen/code/pnpm/packages/pnpm/lib/pnpm.js"`.  And indeed, there was no such file  "/home/glen/code/pnpm/packages/pnpm/lib/pnpm.js".

Only after running `pnpm run compile` once as mentioned as an "alternative" in step 4, would `pd` do anything other than erroring. And indeed, after this run compile, `pd` seemed to function as expected, and seemed to pick up changes to the source code without re-running the `pnpm run compile` step.

So the question is, am I now set up correctly? And should I file an issue (and then presumably a tiny PR) to add a step between the current step 2 and step 3 to run `pnpm run compile` once?

Thanks for the guidance. I felt since I was new to pnpm I should ask here first rather than directly filing an issue.

**Comment by zkochan** [comment, marked as answer]
Feel free to update the docs

**Reply by gwhitney** [reply]
OK, PR #5289 submitted.
